modules/Makefile.kubernetes

```diff
@@ -27,7 +27,13 @@ define kubectl_apply
 	$(call assert,CLUSTER_NAMESPACE)
 	$(call assert,CLUSTER_DOMAIN)
 	@echo -e "INFO: Applying changes to $(KUBERNETES_APP) $(1) on cluster $(call yellow,$(CLUSTER))..."
-	@$(call envsubst,$(KUBERNETES_RESOURCE_PATH)/$(KUBERNETES_APP)-$(1).yml) | $(KUBECTL_CMD) apply $(2) -f -
+	# Optionally load parameters for the kubernetes resources
+	@\
+	set -a; \
```

Comment:
"Each variable or function that is created or modified is given the export attribute and marked for export to the environment of subsequent commands." aka treat the sourced file as export ... variables